src/TextArea/TextArea.tsx

```diff
@@ -225,7 +225,7 @@ export class TextArea extends React.Component<Props, State> {
 
     const textAreaProps = {
       ["aria-invalid"]: !valid,
-      ["aria-label"]: this.props.label,
+      ["aria-label"]: this.props.label || this.props.placeholder,
```

Comment:
For accessibility! Since we may no longer pass a `label` through to the underlying `TextArea` component, let's fallback to using the `placeholder` as the `aria-label`, which will now always be specified